install/linux/docker-ee/rhel.md

```diff
@@ -7,63 +7,50 @@ redirect_from:
 - /engine/installation/linux/rhel/
 - /engine/installation/linux/docker-ee/rhel/
 title: Get Docker EE for Red Hat Enterprise Linux
-toc_max: 4
 ---
 
 {% assign linux-dist = "rhel" %}
+{% assign linux-dist-cap = "RHEL" %}
```

Comment:
You don't really need to define a new variable for this. You can use the existing one and use a filter to capitalize or upcase the value. For that delete this, where you are using this variable, use this instead:

```
{{ linnux-dist | upcase }}
```

[You can learn more here](https://shopify.github.io/liquid/filters/upcase/)

Reply:
Actually, @joaofnfernandes , this solution doesn't work for CentOS and Oracle Linux (OL).